src/components/misc/spinner.tsx

```diff
@@ -1,14 +1,12 @@
+import NavFrontendSpinner from "nav-frontend-spinner";
```

Comment:
trenger du egentlig en egen komponent for denne, kan du ikke bruke `<NavFrontendSpinner />` direkte i `search-bar` ?

Reply:
jo, det har du helt rett i - endret dette i neste commit (Y)